sdk/contentsafety/azure-ai-contentsafety/tests/recordings/test_content_safety.pyTestContentSafetyCasetest_analyze_image.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We moved recordings outside of the repository, here is more information about how to do it: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/recording_migration_guide.md

Reply:
@catalinaperalta Thanks for reviewing my PR! I've resolved most of your comments and we could discuss remaining in our sync. 
I've also updated the sample for blocklist management in sample_manage_blocklist.py, please help review.

Reply:
You can open a tracking issue for this and we can track it separately from this PR

Reply:
@catalinaperalta For this PR, it's allowed to merge code with recordings right? And after that I should migrate recordings.

Reply:
Yes, as long as CI is passing. Just open a tracking issue for this please.

Reply:
@catalinaperalta opened an issue about this: https://github.com/Azure/azure-sdk-for-python/issues/30407, how can I assign this issue to myself?

Reply:
On the right hand side there's an assignees with a gear on the side which you can open and search for your github username, you click on your name and it assigns the issue to you:
<img width="980" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/9859037/016601ab-415a-4bf4-8985-ccdc47cf98c3">